src/sidebar/components/annotation-quote.js

```diff
@@ -43,7 +43,7 @@ function AnnotationQuote({ annotation, settings = {} }) {
           {quote(annotation)}
         </blockquote>
```

Comment:
I'm not sure aside should be used here, but I'm weak on understand the differences.  But I think the only thing that seems a little odd to me is the blockquote nested in the aside. I get the feeling we should use one or the other or maybe rethink this annotation-quote?

Sorry, I'm just being cautious here...

Reply:
Upon further understanding `aside` I think that is the correct tag to use here, so I'm just wondering if we can get rid of the `blockquote` altogether?

Reply:
I think the `blockquote` still has use as it is the actual quote from an external source here.

Reply:
ya, I think I see that aside here is used as a wrapped around the blockquote and the excerpt, that may be a much larger refactor.

Reply:
I'm not sure I understand—I don't think there's a problem here?

The entire segment of content is an `aside`, and the `aside` contains a `blockquote`. I don't _believe_ this is problematic.

Reply:
Do we actually need a semantic wrapper around `blockquote` at all? `aside` doesn't seem right looking at [the MDN example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).

Reply:
It seems like aside can be any sort of content not related to the main content. It does not say its cant wrap other things... so this may be reasonable usage

Reply:
then again, the entire client sidebar is an `aside`

hmmm lol

Reply:
From a practical point of view, one effect of this change is that each quote shows up in the "Landmarks" menu in VoiceOver with as a "complementary". Ordinarily on eg. a news website this is used for a sidebar that presents things like "Most Read Stories" or something.